ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -482,6 +488,21 @@ def test_full_backup_reinstall_restore_with_vault(self):
         """backup, uninstall, reinstall, restore"""
         self._full_backup_restore_with_vault(reinstall=True)
 
+    def test_no_error_message_with_uninstall_ipa_with_kra(self):
```

Comment:
No need to add this test as you already added the check in `_full_backup_restore_with_vault`.

Reply:
@flo-renaud 
Just want to make sure that after restore, this is not seen again during uninstall.

Reply:
ok, makes sense